projects/storefrontapp/server.ts

```diff
@@ -23,6 +23,7 @@ const ssrOptions: SsrOptimizationOptions = {
   timeout: Number(
     process.env['SSR_TIMEOUT'] ?? defaultSsrOptimizationOptions.timeout
   ),
+  cache: process.env['SSR_CACHE'] === 'true', // Enable caching for SSR E2E purposes
```

Comment:
[MINOR]
It sounds like you'd like to enable caching for the purpose of running (all) E2Es.
btw. Do we really need this comment?

Reply:
Correct, the comment could be more precise. But indeed, it's not required. I'd remove it 👌